Tasks/between.js

```diff
@@ -1,18 +1,17 @@
 // Extract substring between prefix and suffix
+'use strict';
 
-getvaluebetween = (str, p, s) => {
-  i = str.indexOf(p)
-    if ( i === -1 ) return ''
-    else {
-      k = i + p.length
-      str =  str.substring(k)
-      if ( s) {
-        i = str.indexOf( s )
-        if (i === -1) {
-          return  '' } else { str =  str.substring(0 , i)
-            }}}
-  return str
-}
+const getValueBetween = (string, prefix, suffix) => {
+  const ERR = -1;
+  let substring = '';
+  const prefixIndex = string.indexOf(prefix);
+  const suffixIndex = string.indexOf(suffix);
+  if (prefixIndex !== ERR && suffixIndex  !== ERR) {
```

Comment:
Use eslint and prettier to fix code style and common conventions

Reply:
I used only eslint.